plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -50,8 +50,7 @@
   public void execute(IProgressMonitor monitor) throws InvocationTargetException, CoreException {
     SubMonitor progress = SubMonitor.convert(monitor, 100);
     
-    // todo just use getproject().getLocationUri()
-    URI location = config.getEclipseProjectLocationUri();
+    URI location = config.getProject().getRawLocationURI();
```

Comment:
Because when I used getLocationUri it didn't work (pressing finish errored out, presumably since it didn't close) and when I swtiched to getRawLocationUri it did. Now that we have better error reporting I can dig a little deeper on this.